sessionctx/variable/session.go

```diff
@@ -1330,6 +1337,8 @@ func (s *SessionVars) SetSystemVar(name string, val string) error {
 		config.GetGlobalConfig().CheckMb4ValueInUTF8 = TiDBOptOn(val)
 	case TiDBFoundInPlanCache:
 		s.FoundInPlanCache = TiDBOptOn(val)
+	case TiDBFoundInSPM:
+		s.FoundInSPM = TiDBOptOn(val)
```

Comment:
Since `FoundInSPM` is just an internal flag, we don't need to support a new variable `Plan_from_SPM` for it.
`SessionVars.DurationParse` is an example.